open-telemetry/otel-collector.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: OpenTelemetry Collector Contrib
+sidebarTitle: OTel Collector Contrib
+---
+
+The following document describes how to natively configure the OpenTelemetry Collector Contrib distribution with the Middleware platform. Use the OpenTelemetry Collector to collect, process, and export your telemetry data that is not accessable through the core OpenTelemetry Project. 
+
+Use the OpenTelemetry Collector Contrib to build your own custom distributions with the [OpenTelemetry Collector Builder](https://opentelemetry.io/docs/collector/custom-collector/), send host data without using the Middleware Host Agent, or send host data from a technology that doesnt have a native Middleware Integration.
```

Comment:
"Use the OpenTelemetry Collector" is used too many times. My .02 is we can probably remove the second sentence of the first paragraph and be good.